packages/core/src/Serialization/CBOR/CborTag.ts

```diff
@@ -48,6 +48,9 @@ export enum CborTag {
   /** Tag value for MIME messages (including all headers), as defined in RFC2045. */
   MimeMessage = 36,
 
+  /** Tag value for `set<a> = #6.258([* a]) / [* a]`, `nonempty_set<a> = #6.258([+ a]) / [+ a]`, `nonempty_oset<a> = #6.258([+ a]) / [+ a]` */
+  Set = 258,
```

Comment:
nitpick, this tag is formally called [MathematicalFinateSet](https://github.com/mofosyne/iana-headers/blob/main/c/src/cbor-constants.h#L184), and it was introduced by IOG 7 years ago to the IANA registry, I wonder by this wasn't initially introduced in the CDDL spec since it was already available

Reply:
Updated in the CborSet type doc